attachments/simple_engine/renderer_rendering.cpp

```diff
@@ -496,597 +510,439 @@ void Renderer::updateUniformBufferInternal(uint32_t currentImage, Entity* entity
     ubo.camPos = glm::vec4(camera->GetPosition(), 1.0f);
 
     // Set PBR parameters (use member variables for UI control)
-    ubo.exposure = this->exposure;
+    // Clamp exposure to a sane range to avoid washout
+    ubo.exposure = std::clamp(this->exposure, 0.2f, 2.0f);
     ubo.gamma = this->gamma;
     ubo.prefilteredCubeMipLevels = 0.0f;
-    ubo.scaleIBLAmbient = 0.5f;
+    ubo.scaleIBLAmbient = 0.25f;
+    ubo.screenDimensions = glm::vec2(swapChainExtent.width, swapChainExtent.height);
+
+    // Signal to the shader whether swapchain is sRGB (1) or not (0) using padding0
+    int outputIsSRGB = (swapChainImageFormat == vk::Format::eR8G8B8A8Srgb ||
+                        swapChainImageFormat == vk::Format::eB8G8R8A8Srgb) ? 1 : 0;
+    ubo.padding0 = outputIsSRGB;
```

Comment:
I think, it's not a good idea to carry some real information in a member called `padding0`